compiler/src/codegen/compcore.re

```diff
@@ -856,16 +854,14 @@ let compile_bind =
     | BindSet(arg) => set_slot(slot, typ, arg)
     | BindTee(arg) => tee_slot(slot, typ, arg)
     };
-  | MGlobalBind(slot, wasm_ty, gc) =>
```

Comment:
Why can we get rid of this? Not 100% sure I understand

Reply:
Turns out we haven't referenced globals by slot # for ages 😅 

I took it out now to avoid making it work with closures given that it's unused.

Reply:
Now I'm more confused 😅 ...the diff is showing that you dropped the `gc` argument, not `slot`

Reply:
Oh sorry ignore me, I was thinking about something totally different. `wasm_ty` implies GC because it tells you if it's `HeapAllocated` etc.

Reply:
Okay, that was sort of what I was thinking, so I just wanted to clarify. Looks good 👍